config.toml

```diff
@@ -1,4 +1,4 @@
-baseurl = "https://intel4coro.github.io/ViB-public/"
+baseurl = "https://srikanth635.github.io/ViB-public/"
```

Comment:
It is not necessary to have the whole domain name in the baseurl.
Please updated it as below to make the config works on both repos.
```
baseurl = "ViB-public/"
```